### PR TITLE
Major css changes

### PR DESCRIPTION
**Pull request by AnjaniKumar1515**
Major css changes in the website. 
Added new components. 
@Chandan-Gowda-Kalvium helped with the css of navbar and searcbox.

### CODE REVIEW

File: frontend/src/Components/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import "../App.css";
+import Typewriter from "typewriter-effect";
+
+function SearchBar() {
+  const [search, setSearch] = useState("");
+  const [FundsData, setFundsData] = useState(null);
+  const [fund, setFund] = useState(null);
+
+  useEffect(() => {
+    axios.get("http://localhost:8000/apis/v1/funds").then((resp) => {
```
Comment (PatelNeelMahesh) [inline frontend/src/Components/SearchBar.js:15]: Add `http://localhost:8000` to `.env` and use `process.env` here.

File: frontend/src/Components/Explore.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import "./Explore.css";
+
+const Explore = () => {
+  const [activeFunds, setActiveFunds] = useState([]);
+  const [orderBy, setOrderBy] = useState("");
+  const [order, setOrder] = useState("asc");
+  const [searchText, setSearchText] = useState("");
+
+  const handleClick = async (e) => {
+    await axios.get("http://localhost:8000/apis/v1/funds").then((resp) => {
```
Comment (PatelNeelMahesh) [inline frontend/src/Components/Explore.js:20]: Add `http://localhost:8000` to `.env` and use `process.env` here, later when you deploy backend you only have to do changes in one file.

**Review by PatelNeelMahesh — COMMENTED** [review]
Do required changes before merging pull request.

File: frontend/src/Components/Explore.js (hunk 2 above)
Comment (AnjaniKumar1515) [reply frontend/src/Components/Explore.js:20]: Okay, I will apply the changes and merge.

**Pull request merged**